### PR TITLE
AP_Motors: add DCP trim feature for Dual Heli

### PR DESCRIPTION
**Pull request by bnsgeyer**
This PR adds the ability to trim the differential collective pitch (DCP) or the relative height of the swashplates for dual heli configurations. It was requested so it would be easier to trim out CG offsets.  The parameter added corresponds to the DCP axis (pitch for tandem or roll for transverse) input to the motors class that is need to maintain a hover in trim for the axis that uses DCP.  So if the integrator is holding 0.05 in a zero wind hover then the DCP trim parameter would be set to 0.05 to zero out the integrator.

This feature was tested in SITL by simulating a cg offset in the dual-heli sim.

### CODE REVIEW

**Review by tridge — APPROVED** [review]

**Review by rmackay9 — APPROVED** [review]

File: libraries/AP_Motors/AP_MotorsHeli_Dual.cpp [hunk 1]
```diff
@@ -196,6 +196,14 @@ const AP_Param::GroupInfo AP_MotorsHeli_Dual::var_info[] = {
     // @Increment: 1
     AP_SUBGROUPINFO(_swashplate2, "SW2_", 21, AP_MotorsHeli_Dual, AP_MotorsHeli_Swash),
 
+    // @Param: DCP_TRIM
+    // @DisplayName: Differential Collective Pitch Trim
+    // @Description: Used to trim DCP axis.  DCP_TRIM has the same units as the axis commanded output.  A value of 1 is full control output.  Trim is limited to 20 percent of output in one direction.
```
Comment (rmackay9) [inline libraries/AP_Motors/AP_MotorsHeli_Dual.cpp:201]: As brought up by @Naterater perhaps a year ago, we should avoid putting "Used to" or "This Controls", etc at the beginning of a parameter description because all parameters are used to do something or control something. We can just skip right to the point and say, "DCP Trim.." or something like that

File: libraries/AP_Motors/AP_MotorsHeli_Dual.cpp [hunk 2]
```diff
@@ -196,6 +196,14 @@ const AP_Param::GroupInfo AP_MotorsHeli_Dual::var_info[] = {
     // @Increment: 1
     AP_SUBGROUPINFO(_swashplate2, "SW2_", 21, AP_MotorsHeli_Dual, AP_MotorsHeli_Swash),
 
+    // @Param: DCP_TRIM
+    // @DisplayName: Differential Collective Pitch Trim
+    // @Description: Removes I term bias due to center of gravity offsets or discrepancies between rotors in swashplate setup.
```
Comment (rmackay9) [inline libraries/AP_Motors/AP_MotorsHeli_Dual.cpp:201]: I wonder if a better description might be to describe the physical difference it makes first.  For example negative numbers lower the 2nd swash vs the 1st.

Comment (bnsgeyer) [reply libraries/AP_Motors/AP_MotorsHeli_Dual.cpp:201]: @rmackay9  I considered that but thought it might get too wordy.  Would it make sense to say as a second sentence.  “if DCP axis has I term bias while hovering in calm winds, use value of bias in DCP_TRIM to re-center I term.”

Comment (rmackay9) [reply libraries/AP_Motors/AP_MotorsHeli_Dual.cpp:201]: ok, sounds good to me!
